### DISCUSSION TITLE
Export recipes to plain MD wiki like repo

### DISCUSSION THREAD

**Discussion by Thijzer** [mealie-recipes/mealie#3615] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I've been chewing on this idea a lot. 
Definitely not the meat of what Mealie is all about.
But I do recognize from my own experiences is that after the hard work, I no longer need the managing bit. 
It's just a plain and simple RO repo for me to search my recipes.
Just like a good Hugo theme or even a wiki an github.

Before I build this, just wanted to know if nothing is in the works already, and do people agree  for this feature or middle-ware...
 
I would build this feature myself, it's isolated enough to do so..

### Please provide a concise description of the feature that would resolve your issue.

all above

### Please consider and list out some caveats or tradeoffs made in your design decision

all above

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Mealie has always been about being in control of your own data, so a markdown export is certainly a reasonable feature. There's currently a download feature with the same idea in mind (though it's broken, see https://github.com/mealie-recipes/mealie/pull/3579)
